extensions/shared/components/audio-player/index.jsx

```diff
@@ -0,0 +1,83 @@
+/* global _wpmejsSettings, MediaElementPlayer */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { speak } from '@wordpress/a11y';
+
+/**
+ * Internal dependencies
+ */
+import { STATE_PLAYING, STATE_PAUSED, STATE_ERROR } from './constants';
+
+const meJsSettings = typeof _wpmejsSettings !== 'undefined' ? _wpmejsSettings : {};
+
+function AudioPlayer( {
+	trackSource,
+	handlePlay,
+	handlePause,
+	handleError,
```

Comment:
Should these properties be renamed as events? because they are events :-)

```suggestion
	onPlay,
	onPause,
	onError,
```